common/exprParser/helpers/sqlUtils.js

```diff
@@ -0,0 +1,54 @@
+const { types } = require('./types')
+
+const getSqlOperator = op => {
+  switch (op) {
+  case '||':
+    return 'OR'
+  case '&&':
+    return 'AND'
+  case '===':
+    return '='
+  case '!==':
+    return '!='
+  default:
+    throw new Error(`Unknown operator: ${op}`)
+  }
+}
+
```

Comment:
I wouldn't use a switch. but a simple key value object. e.g. {'||':'OR' , '&&' : 'AND' }